example/swift/example.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:ffi';
+import 'swift_api_bindings.dart';
+
+void main() {
```

Comment:
Please add a TODO to run this as test on the CI with a reference to the blocking issue.